litebird_sim/observations.py

```diff
@@ -96,7 +96,7 @@ def __init__(
             self._n_blocks_time = 1
 
         if allocate_tod:
-            self.tod = np.empty(
+            self.tod = np.zeros(
```

Comment:
I like this change: it makes everything simpler from the point of view of the user.